### ISSUE TITLE
fwrite() errror on DBClose()

### ISSUE THREAD

**Issue by MrMitch** [AntonTerekhov/OrientDB-PHP#15]
Hi,

I regulary get a `fwrite()` error when the `DBClose()` command is called. Although it doesn't seem to prevent the application to write or read from OrientDB, the error is always reported.
Here is a stack trace example:

```
Fatal error: Uncaught exception 'ErrorException' with message 'fwrite(): 85 is not a valid stream resource' in /home/mitch/www/app/orientdb/OrientDBSocket.php:88

Stack trace:
#0 [internal function]: framework\errorHandler\ErrorHandler->errorHandler(2, 'fwrite(): 85 is...', '/home/mitch/www...', 88, Array)
#1 /home/mitch/www/app/orientdb/OrientDBSocket.php(88): fwrite(Resource id #85, '?????')
#2 /home/mitch/www/app/orientdb/Commands/OrientDBCommandAbstract.php(174): OrientDBSocket->send('?????')
#3 /home/mitch/www/app/orientdb/OrientDB.php(299): OrientDBCommandAbstract->execute()
#4 /home/mitch/www/app/orientdb/OrientDB.php(256): OrientDB->__call('DBClose', Array)
#5 /home/mitch/www/app/orientdb/OrientDBSocket.php on line 88
```

**Comment by AntonTerekhov** [comment]
Hi!

Could you please be more specific. PHP version, OrientDB version, this library version and small code snippet.
Something similiar was fixed with revision 0343357 in issue #12 .

**Comment by MrMitch** [comment]
Perfect, 6ef9952de5141fdd2fad99e656877d3e019491be solved this issue. I think the issue comes from the fact that OrientDB doesn't properly close the data files.
